sql/data-types/timestamp.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Timestamp"
+sidebarTitle: Timestamp type
+description: "Use the timestamp data type to handle date and time information."
+---
+
+Timestamp data types are used to store and process precise time information. RisingWave supports two types of timestamps:
+
+- `timestamp`: Stores date and time values without timezone, supporting up to **nanosecond** precision (9 digits after the decimal point).
+
+- `timestamptz`: Stores date and time values along with time zone information, supporting up to **microsecond** precision (6 digits after the decimal point).
+
+
+## Precision preservation
+
+<Note>
+Added in version 2.1.
```

Comment:
https://www.notion.so/Working-with-Nanosecond-Precision-Timestamps-in-RisingWave-draft-1e2f0d69cb7680dbafa1ec51842354ad?d=1e3f0d69cb7680e08b7f001c1ee500ef&pvs=4#1e3f0d69cb7680b8a436ee5287f0ed27

The nano precision in v2.1 is problematic and will be REVERTED for a better alternative (without the precision loss issue below). Let's refrain from publishing it.

Reply:
The proper nano precision support is not even being worked on in v2.5. So the "promotion" of nano support shall be put on hold until it is ready.

If we are going to have a dedicated page for time related data types, it is better focus on the difference between timestamp and timestamptz, and when to use which (short answer: always prefer timestamptz unless you stick to a single offset)